build.zig

```diff
@@ -464,6 +470,24 @@ fn git_commit(allocator: std.mem.Allocator) ?[40]u8 {
     return output;
 }
 
+fn git_tag(allocator: std.mem.Allocator) ?[100]u8 {
```

Comment:
Oh, this needs to be `?std.BoundedArray(u8, 100)`.

Reply:
Yup.